p7.py

```diff
@@ -1,3 +1,4 @@
+#start
 #https://leetcode.com/problems/search-insert-position/
 #visit the above link to see problem  description
 class Solution:

```

Comment:
Apply binary search and do it in O(logn) time
When number is present, it is straight forward.
When number is not present, the value of lower bound will give you the correct answer (Since array does not have duplicates )